tests/utils.py

```diff
@@ -168,6 +168,18 @@ def assert_function_in_collapsed(function_name: str, collapsed: StackToSampleCou
     assert is_function_in_collapsed(function_name, collapsed), f"function {function_name!r} missing in collapsed data!"
 
 
+def assert_ldd_version_container(container: Container, version: str) -> None:
+    pattern = r"^ldd \(.*\) (\d*.\d*)"
+    command = "ldd --version"
+    exec_output = container.exec_run(command).output.decode("utf-8")
+    search_result = re.search(pattern, exec_output)
+    if search_result:
+        version_in_container = search_result.group(1)
+    else:
+        version_in_container = False
```

Comment:
Assigning a string or a boolean is not okay (I think mypy should complain on it o.O). It does work because, well, `False` will always be different from the version.

It's better to assign `None` if you couldn't find what you need, the comparison will fail as well